cmake/projects/libbacktrace/hunter.cmake

```diff
@@ -0,0 +1,42 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+#include(hunter_cmake_args)
+include(hunter_configuration_types)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    libbacktrace
+    VERSION
+    1.0.0-ca0de051
+    URL
+    "https://github.com/ianlancetaylor/libbacktrace/archive/ca0de0517f3be44fedf5a2c01cfaf6437d4cae68.tar.gz"
+    SHA1
+    7c7db4d096d96f329c453fb97b7a456fe823e943
+)
+
+#if (ANDROID OR IOS)
```

Comment:
Remove if unused

Reply:
Arg, I always forget this ... thanks for the hint :+1:  ... I removed it.